.github/workflows/build.yml

```diff
@@ -11,8 +11,7 @@ on:
   workflow_dispatch:
 
 env:
-  _AZ_REGISTRY: 'bitwardenprod.azurecr.io'
-
+  _AZ_REGISTRY: "bitwardenprod.azurecr.io"
```

Comment:
ℹ️ Prettier is gonna remove this extra newline. I have heard our workflow linter or at least our "rules" want two lines here but that can't be configured by Prettier.

Reply:
Thanks. The double lines might actually have been a mistake.